llvm/docs/SPIRVUsage.rst

```diff
@@ -237,6 +237,8 @@ Below is a list of supported SPIR-V extensions, sorted alphabetically by their e
      - Adds predicated load and store instructions that conditionally read from or write to memory based on a boolean predicate.
    * - ``SPV_KHR_maximal_reconvergence``
      - Adds execution mode and capability to enable maximal reconvergence.
+   * - ``SPV_INTEL_blocking_pipes``
```

Comment:
Future of Intel FPGA extensions was a bit uncertain. Now it's more clear - they are renamed to Altera extensions in https://github.com/KhronosGroup/SPIRV-Registry/commit/e86c766cfbff190912b48c6bddfec6292835b81a . I suggest to start generating new names for the extensions, capabilities and instructions.